### DISCUSSION TITLE
再接続時の挙動について

### DISCUSSION THREAD

**Discussion by coffeebreak0811** [Yowkees/keyball#315] [Q&A] [answered]
macに接続して使っています
PC再起動時またはUSB再接続時に「keyball」と画面に表示されキーボードとして認識しなくなります
Remapを使用してKeyball61_viaを書き込むと前回のキー割当て通りに使えるようになります

つまり毎回PC起動後にRemapを使用しないと使えない状態です
皆さんのコメントを拝見していると接続するだけで使えるようですが何か設定間違えやハンダミスなどあるのでしょうか？
お手数ですがアドバイス頂けますと助かります

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

両手ともにKeyballロゴが表示される場合は正しくUSBデバイスとして認識されていない状態です。
お使いのPC環境によって起こる現象で、USBケーブルを抜き差しすると解消されます。

頻繁に抜き差しが発生する場合には、マグネット接続タイプのUSBケーブルをご使用いただくと、ProMicroのコネクタもげ対策にもなって良いかと思います。
お手数ですがご検討お願いいたします。

**Reply by coffeebreak0811** [reply]
ありがとうございます
検討します
